### DISCUSSION TITLE
'"mysqldump"' n’est pas reconnu en tant que commande interne ou externe

### DISCUSSION THREAD

**Discussion by MedRiadhKhalfallah** [spatie/laravel-backup#1257] [Q&A] [answered]
i'm using php 7.3 with laravel 8.8.0 and laravel-backup 6.15.0
and i get this err:
Backup failed because: The dump process failed with exitcode 1 : General error : '"mysqldump"' n’est pas reconnu en tant que commande interne ou externe, un programme exécutable ou un fichier de commandes.

**Comment by fredmorinmibu** [comment, marked as answer]
I had same problem.  Using MariaDb probably. make sure to add the dump part between port and username (it can be anywhere inside the mysql config part in the database.php file.  Make sure you replace my binary_path by yours

      'port' => env('DB_PORT', '3306'),
            'dump' => [
                'dump_binary_path' => 'C:\\Program Files\\MariaDB 10.5\\bin\\', // only the path, so without `mysqldump` or `pg_dump`
                'use_single_transaction',
                'timeout' => 60 * 5, // 5 minute timeout
            ],      'database' => env('DB_DATABASE', 'forge'),
            'username' => env('DB_USERNAME', 'forge'),
